pkg/xen-tools/xen-start

```diff
@@ -46,7 +46,10 @@ echo 0 > /dev/unknownStateCounter
 (while true; do
    sleep 11
    /etc/xen/scripts/xen-info "$1" > "/run/tasks/$1.tmp"
-   mv "/run/tasks/$1.tmp" "/run/tasks/$1"
+   if [ -s "/run/tasks/$1.tmp" ]
```

Comment:
I don't understand this logic -- the right thing to do is to fix whatever may be broken in xen-info

Reply:
Removed. 
Will fix this in a separate PR after investigating the cause.